app/components/dashboard_course_component/dashboard_course_component.scss

```diff
@@ -5,43 +5,41 @@
   border-left: 10px solid $lime-green;
 }
 
-.dashboard-course-component__wrapper {
+.dashboard-course-component__top-wrapper {
   display: flex;
   gap: 20px;
   margin-top: 14px;
+  margin-bottom: 14px;
 
   @include govuk-media-query($until: desktop) {
     flex-direction: column;
   }
-}
 
-.dashboard-course-component__programme {
-  flex: 1;
+  .dashboard-course-component__programmes {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+  }
 
-  &-wrapper {
-    margin-bottom: 12px;
+  .ncce-activity-list__tags {
+    margin: 0;
+  }
 
-    p {
-      display: inline-block;
-      background-color: $light-green;
-      padding: 5px 10px;
-      border-radius: 5px;
-      text-wrap: balance;
-      margin-bottom: 0;
-    }
+  .dashboard-course-component__format {
+    display: flex;
+    padding-top: 5px;
 
-    &:last-child {
-      margin-bottom: 0;
+    span {
+      font-size: 16px;
```

Comment:
You could use the govuk mixin here to set this, as it brings in other settings related to typography.

`@include govuk-font($size: 16);`

Reply:
Good spot! Will update this.